src/ducks/reimbursements/Reimbursements.jsx

```diff
@@ -1,192 +1,139 @@
+              showTriggerErrors={false}
+            />
+          ) : (
+            <>
+              <Divider />
+              <NoPendingReimbursements
+                period={formattedPeriod}
+                doc={filteringDoc}
               />
-            ) : (
-              <>
-                <Divider />
-                <NoPendingReimbursements
-                  period={formattedPeriod}
-                  doc={filteringDoc}
-                />
-              </>
-            )}
-          </Section>
-          <Section>
-            <SectionTitle>{t('Reimbursements.alreadyReimbursed')}</SectionTitle>
-            {hasReimbursedExpenses ? (
-              <TransactionList
-                transactions={reimbursedExpenses}
-                className={styles.Reimbursements__transactionsList}
-                showTriggerErrors={false}
+            </>
+          )}
+        </Section>
+        <Section>
+          <SectionTitle>{t('Reimbursements.alreadyReimbursed')}</SectionTitle>
+          {hasReimbursedExpenses ? (
+            <TransactionList
+              transactions={reimbursedExpenses}
+              className={styles.Reimbursements__transactionsList}
+              showTriggerErrors={false}
+            />
+          ) : (
+            <>
+              <Divider />
+              <NoReimbursedExpenses
+                hasHealthBrands={hasHealthBrands}
+                doc={filteringDoc}
               />
-            ) : (
-              <>
-                <Divider />
-                <NoReimbursedExpenses
-                  hasHealthBrands={hasHealthBrands}
-                  doc={filteringDoc}
-                />
-              </>
-            )}
-          </Section>
-        </div>
-      </TransactionActionsProvider>
-    )
-  }
+            </>
+          )}
+        </Section>
+      </div>
+    </TransactionActionsProvider>
+  )
 }
 
-function mapStateToProps(state) {
-  return {
-    groupedExpenses: getGroupedFilteredExpenses(state),
-    currentPeriod: getPeriod(state),
-    filteringDoc: getFilteringDoc(state)
-  }
-}
+const Reimbursements = props => {
+  const { addFilterByPeriod } = useFilters()
+  const groupedExpenses = useSelector(getGroupedFilteredExpenses)
+  const currentPeriod = useSelector(getPeriod)
+  const filteringDoc = useSelector(getFilteringDoc)
+  const transactions = useQuery(transactionsConn.query, transactionsConn)
+  const { brands } = useBrands({ queryName: 'reimbursementsPageTriggers' }) // We need to have a different query name otherwise we end with an infinite loading
 
-const Reimbursements = compose(
-  translate(),
-  queryConnect({
-    transactions: transactionsConn
-  }),
-  connect(mapStateToProps),
-  withFilters,
-  // We need to have a different query name otherwise we end with an infinite
-  // loading
-  withBrands({ queryName: 'reimbursementsPageTriggers' })
-)(DumbReimbursements)
+  return (
+    <DumbReimbursements
+      addFilterByPeriod={addFilterByPeriod}
```

Comment:
this component DubmReimbursements looks useless. Let's do the operation directly inside Reimbursement.

Reply:
comme je disais dans la description de la PR : J'ai gardé l'idée du DumbReimbursements pour éviter de refactor tous les tests.